File: 06.PHP/pg-226-230/pg_226_03.php
<?php
echo("");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHP<br>");
echo("READ AN INTEGER AND DISPLAY ALL ITS EXACT DIVISORS BETWEEN 1 AND THE GIVEN NUMBER<br>");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHP<br><br>");
echo("");

$i =1;
echo("Write an integer: ");
$num = 18;
echo($num . "<br>");

if($num < 0)
    {
        $num *= -1;
    }
/*
while($i <= $num)
    {
        if($num % $i == 0)
            {
                echo($i. ", ");
            }
    $i++;
    }
*/

for($i; $i <= $num; $i++)
    {
        if($num % $i == 0)
            {
                echo($i . ", ");
            }
    }


?>

File: 02.PYTHON/pg-226-230/pg_226_03.py
print("");
print("PYPYPPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYYPYPYPYPYPYPYPYYPYPYPY");
print("READ AN INTEGER AND DISPLAY ALL ITS EXACT DIVISORS BETWEEN 1 AND THE GIVEN NUMBER");
print("PYPYPPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYYPYPYPYPYPYPYPYYPYPYPY");
print("");

i = 1;
num = int(input("Write an integr: "));

if(num < 0):
    num *= -1;
"""
while(i <= num):

    if(num % i == 0):
        print(i);
    i += 1;
"""

for i in range(1, num+1, 1):
    if(num % i == 0):
        print(i);